ovh-redirections

```diff
@@ -71,15 +71,27 @@ def fetch():
 
     with open(remote_db, 'w') as f_r:
         yaml.dump(redirections_dict, f_r, indent=2, default_flow_style=False)
+
     print('Redirections written to "{}"'.format(remote_db))
 
+    # Try to update local IDs if the local database exists
+    if os.path.isfile(local_db):
+        _update_local_ids()
+    else:
+        print("Local database not found, skipping update local IDs...")
```

Comment:
You should remove this displayed text about "not updating ID" : its more a debug message than user message.